internal/static/traces/spanmetricsprocessor/config.go

```diff
@@ -0,0 +1,122 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
I take it we forked the spanmetricsprocessor to keep it in the codebase? 

Since this code never runs in Flow mode, have we considered _only_ including the config types and having the factory be a no-op? That way the converters still work but minimize the amount of code we introduce.

Reply:
I forked the component because if we upgrade Agent and static mode, we are going to have to make it work. In Alloy it's just used for converters. I agree that in the future we should remove all static mode code from Alloy, and we could just keep the configs for so that the converters can use them.